samples/dml.js

```diff
@@ -529,6 +529,61 @@ async function deleteUsingPartitionedDml(instanceId, databaseId, projectId) {
   // [END spanner_dml_partitioned_delete]
 }
 
+function updateUsingBatchDml(instanceId, databaseId, projectId) {
+  // [START spanner_dml_batch_update]
+  // Imports the Google Cloud client library
+  const {Spanner} = require('@google-cloud/spanner');
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
+  // Gets a reference to a Cloud Spanner instance and database
+  const instance = spanner.instance(instanceId);
+  const database = instance.database(databaseId);
+
+  const insert = {
+    sql: `INSERT INTO Albums (SingerId, AlbumId, AlbumTitle, MarketingBudget)
+      VALUES (1, 3, "Test Album Title", 10000)`,
+  };
+
+  const update = {
+    sql: `UPDATE Albums SET MarketingBudget = MarketingBudget * 2
+      WHERE SingerId = 1 and AlbumId = 3`,
+  };
+
+  const dmlStatements = [insert, update];
+
+  database.runTransaction(async (err, transaction) => {
```

Comment:
can we use `async/await` here?

Reply:
Here's an example of that - https://github.com/googleapis/nodejs-spanner/blob/master/system-test/spanner.ts#L3836